src/compressed_chunk.c

```diff
@@ -59,8 +59,8 @@ static void trimChunk(CompressedChunk *chunk, int minSize) {
 
     assert(excess >= 0); // else we have written beyond allocated memory
 
-    if (excess > 0) {
-        size_t newSize = max(chunk->size - excess + 2, minSize);
+    if (excess > 1) {
```

Comment:
@ashtul i would move the +1 to the excess calculation and not drag it along the function